op-conductor/conductor/flashblocks_handler.go

```diff
@@ -0,0 +1,209 @@
+package conductor
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/gorilla/websocket"
+)
+
+// StartFlashblocksHandler initializes the flashblocks handler and starts the rollup boost listener
+func (oc *OpConductor) StartFlashblocksHandler(ctx context.Context) error {
+	// Start the WebSocket server
+	if err := oc.startWebSocketServer(ctx); err != nil {
+		return err
+	}
+
+	// Start the rollup boost listener
+	if err := oc.startRollupBoostListener(ctx); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+// startRollupBoostListener initializes and starts a WebSocket client to listen for rollup boost messages
+func (oc *OpConductor) startRollupBoostListener(_ context.Context) error {
+	if oc.cfg.RollupBoostWsURL == "" {
+		oc.log.Info("rollup boost WebSocket disabled, no URL configured")
+		return nil
+	}
+
+	oc.log.Info("connecting to rollup boost WebSocket", "url", oc.cfg.RollupBoostWsURL)
+
+	// Connect to the rollup boost WebSocket
+	conn, _, err := websocket.DefaultDialer.Dial(oc.cfg.RollupBoostWsURL, nil)
+	if err != nil {
+		return fmt.Errorf("failed to connect to rollup boost WebSocket: %w", err)
+	}
+
+	oc.rollupBoostConn = conn
+	oc.log.Info("connected to rollup boost WebSocket", "url", oc.cfg.RollupBoostWsURL)
+
+	// Start a goroutine to read messages from the rollup boost WebSocket
+	go func() {
+		defer func() {
+			oc.log.Info("closing rollup boost WebSocket connection")
+			conn.Close()
+			oc.rollupBoostConn = nil
+		}()
+
+		for {
+			_, message, err := conn.ReadMessage()
+			if err != nil {
+				oc.log.Error("error reading from rollup boost WebSocket", "err", err)
+				return
+			}
+
+			oc.handleRollupBoostMessage(message)
+		}
+	}()
+
+	return nil
+}
+
+// handleRollupBoostMessage processes a message received from rollup boost
+func (oc *OpConductor) handleRollupBoostMessage(message []byte) {
+	// Only forward messages if we're the leader
+	if !oc.leader.Load() {
+		oc.log.Debug("not forwarding rollup boost message, not the leader")
+		return
+	}
+
+	// Forward the message to connected clients
+	oc.broadcastToClients(message)
+}
+
+// startWebSocketServer initializes and starts a WebSocket server
+func (oc *OpConductor) startWebSocketServer(ctx context.Context) error {
```

Comment:
checkout how other services are initialized, reference `initHealthMonitor` for a more consistent style